mlflow/genai/judges/utils.py

```diff
@@ -84,6 +142,185 @@ def _sanitize_justification(justification: str) -> str:
     return justification.replace("Let's think step by step. ", "")
 
 
+def _split_messages_for_databricks(messages: list["ChatMessage"]) -> tuple[str | None, str]:
+    """
+    Split a list of ChatMessage objects into system and user prompts for Databricks API.
+
+    Args:
+        messages: List of ChatMessage objects to split.
+
+    Returns:
+        Tuple of (system_prompt, user_prompt) where system_prompt may be None.
+
+    Raises:
+        MlflowException: If the messages list is empty or invalid.
+    """
+    from mlflow.types.llm import ChatMessage
+
+    if not messages:
+        raise MlflowException(
+            "Invalid prompt format: expected non-empty list of ChatMessage",
+            error_code=BAD_REQUEST,
+        )
+
+    system_prompt = None
+    user_parts = []
+
+    for msg in messages:
+        if isinstance(msg, ChatMessage):
+            if msg.role == "system":
+                if system_prompt is None:
+                    system_prompt = msg.content
+                else:
+                    user_parts.append(f"System: {msg.content}")
```

Comment:
Can we add an inline comment explaining why we only use the first system message directly in the system prompt? (ordering)